controller/user.controller.js

```diff
@@ -43,14 +42,11 @@ const insertExperiment = async (uuid, experiment) => {
   return updatedUser;
 };
 
-const getUserExperiment = (user, experimentId) => {
-  if (!user[0].experiments) {
+const getUserExperiment = (user) => {
```

Comment:
what is the reson for passin an aray of users and not a single user if anyway we treat only the user[0]

Reply:
You are right, see above